static/js/components/Payment.js

```diff
@@ -28,7 +29,8 @@ export default class Payment extends React.Component<*, void> {
     sendPayment: () => void,
     setPaymentAmount: (event: InputEvent) => void,
     setSelectedBootcampRunKey: (event: InputEvent) => void,
-    showDialog: (dialogKey: string) => void
+    showDialog: (dialogKey: string) => void,
+    currentUser: () => Promise<HttpAuthResponse<User>>
```

Comment:
I think this should just be `?CurrentUser` since it's there's no `Promise` being resolved and the code below is using the object directly and not via `.body`